src/components/ApplicationViews.js

```diff
@@ -1,46 +1,26 @@
-import React, { Component } from "react";
-import { Route, Redirect } from "react-router-dom"
-import Login from "./auth/Login";
-import CreateAccount from "./auth/CreateAccount";
-import Home from "./songs/SongCollection";
-import SongView from "./songs/Song";
+import React, { Fragment } from 'react'
+import { Route, Redirect } from 'react-router-dom'
+import Login from './auth/Login'
+import { CreateAccount } from './auth/CreateAccount'
+import Home from './songs/SongCollection'
+import SongView from './songs/Song'
 
-class ApplicationViews extends Component {
+const ApplicationViews = ({ isAuthenticated, setUser, ...props }) => (
+  //If a user is logged in then render Home, otherwise render Login
+  <Fragment>
```

Comment:
Personal preference... but I don't like to see `React` all over the code.  It also makes things a little "DRY"er to just import `Fragment` from react so that you can just write `Fragment` instead of `React.Fragment` ... this comes in handy with a _TON_ of react stuff.